src/installer/corehost/cli/hostpolicy/hostpolicy_context.cpp

```diff
@@ -42,6 +42,32 @@ namespace
 
         return bundle::runner_t::app()->probe(file_path, offset, size);
     }
+
+#if defined(NATIVE_LIBS_EMBEDDED)
+    extern "C" const void* GlobalizationResolveDllImport(const char* name);
+    extern "C" const void* CompressionResolveDllImport(const char* name);
+
+    // pinvoke_override:
+    // Check if given function belongs to one of statically linked libraries and return a pointer if found.
+    const void* STDMETHODCALLTYPE pinvoke_override(const char* libraryName, const char* entrypointName)
+    {
+        if (strcmp(libraryName, "libSystem.Globalization.Native") == 0)
```

Comment:
Do we need this here? It should be covered by CoreCLR DefaultResolveDllImport.

Reply:
Right, now that default override always runs (not only when host does not provide anything), we do not need this here. 

From one point - static host does link Globalization so it looks like it needs to override, from another - it has to link because coreclr has a default override that calls into Globalization and the whole thing would not build if Globalization is not linked.